### DISCUSSION TITLE
Formily现在提供JsonSchema转jsx或在tsx能力嘛，非markup那种

### DISCUSSION THREAD

**Discussion by dingyanqiang** [alibaba/formily#1836] [Q&A 2.x] [answered]
Formily现在提供JsonSchema转jsx或在tsx能力嘛，非markup那种；
或者有没有计划支持JsonSchema转出jsx
非常喜欢Formily，这个问题对我们影响很大，而且很急，麻烦各位老板尽快解答，谢谢！

**Comment by janryWang** [comment, marked as answer]
不支持，因为纯jsx是不具备协议驱动能力的

**Reply by dingyanqiang** [reply]
收到   谢谢
